sdk/test/opentelemetry/sdk/trace/export/simple_span_processor_test.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+def stub_span_builder(sampled: false)
```

Comment:
Minor quibble: replace "sampled" with "recording" here and throughout this file.